### PR TITLE
Add travis-ci support

### PR DESCRIPTION
**Pull request by LionOps**

### CODE REVIEW

**Comment by LionOps** [comment]
Decided not to add it.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: README.md [hunk 1]
```diff
@@ -124,5 +127,10 @@ The `error` and `page` keys are set by Pixabayjs for convenience. When an error
 ## License
 MIT
 
+<<<<<<< HEAD
```
Comment (kahnvex) [inline README.md]: Bad commit.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by kahnvex** [comment]
:+1:

**Pull request merged**